packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -210,6 +245,31 @@ export class ProjectDataSource {
     this._specWatcher.on('unlink', onSpecsChanged)
   }
 
+  async defaultSpecFileName () {
+    const defaultFileName = 'cypress/e2e/filename.cy.js'
+
+    try {
+      if (!this.ctx.currentProject || !this.ctx.coreData.currentTestingType) {
+        return null
+      }
+
+      let specPatternSet: string | undefined
+      const { specPattern = [] } = await this.ctx.project.specPatternsForTestingType(this.ctx.currentProject, this.ctx.coreData.currentTestingType)
+
+      if (Array.isArray(specPattern)) {
+        specPatternSet = specPattern[0]
```

Comment:
We should make a notes in the documentation that the first pattern provided in the specPattern configuration will be used to generate a spec file from the Cypress UI.

cc/ @cowboy

Reply:
Is there an explanation of how the pattern will be used to generate a spec file name?